invoices/test_utils_test.go

```diff
@@ -255,7 +255,9 @@ func timeout() func() {
 
 	go func() {
 		select {
-		case <-time.After(10 * time.Second):
+		// Use a longer timeout to accommodate slow Postgres database
+		// setup and migrations when running tests in parallel.
+		case <-time.After(120 * time.Second):
```

Comment:
please move the flake fixes to a separate PR, so we can merge them quickly. Moreover they are not related I saw them in other CI runs as well.